tests/unit/#183.test.scala

```diff
@@ -0,0 +1,76 @@
+// SYNTAX TEST "source.scala"
+
+
+    s"\\"
+//  ^ source.scala keyword.interpolation.scala
+//   ^ string.quoted.double.interpolated.scala punctuation.definition.string.begin.scala
+//    ^^ constant.character.escape.scala
+//      ^ punctuation.definition.string.end.scala
+
+    raw"\\"
+//  ^^^ source.scala keyword.interpolation.scala
+//     ^ string.quoted.double.interpolated.scala punctuation.definition.string.begin.scala
+//      ^^ string.quoted.double.interpolated.scala
+//      ^^ - constant.character.escape.scala
+//        ^ punctuation.definition.string.end.scala
+   
+    raw"$$ " // `$"` is an escaped `"` in raw interpolators
+//  ^^^ source.scala keyword.interpolation.scala
+//     ^ string.quoted.double.interpolated.scala punctuation.definition.string.begin.scala
+//      ^^ constant.character.escape.scala
+//        ^ string.quoted.double.interpolated.scala
+//         ^ punctuation.definition.string.end.scala
+
+    raw"$" " // `$$` is an escaped `$` in raw interpolators
```

Comment:
I think the comments are switched between this example and the previous one

Reply:
Indeed